src/openms/source/CHEMISTRY/EmpiricalFormula.cpp

```diff
@@ -206,8 +206,7 @@ namespace OpenMS
     IsotopeDistribution comp_fragment_isotope_dist = complementary_fragment.getIsotopeDistribution(CoarseIsotopePatternGenerator(max_depth));
 
     IsotopeDistribution result;
-    CoarseIsotopePatternGenerator solver;
-    result = solver.calcFragmentIsotopeDist(fragment_isotope_dist, comp_fragment_isotope_dist, precursor_isotopes);
```

Comment:
is this simply to improve performance and not create a new object each time?

Reply:
The solver is now a parameter to the function and therefore doesn't need to be created inside the function. This way you can choose if you want rounded masses or not.